tekton/pipeline-templates/params/common.yml.j2

```diff
@@ -0,0 +1,81 @@
+# CLI Version
+- name: cli_version
+  type: string
+  description: version of mas cli to be used in tasks and dependencies
+  default: ""
+
+# Registry
```

Comment:
`common` to me implies there are params used in every pipeline, but these `fvt_` params would only be used in the fvt pipelines, not the "normal" ones ... am I misunderstanding the context of "common" here?

Reply:
Here is what happen @durera :

When upgrade-test pipeline runs other pipelines, it will be running, in the end, the run-pipeline role  (this one [here](https://github.ibm.com/maximoappsuite/mas-tekton/tree/master/ansible/ibm/mas_tekton/roles/run_pipeline))  inside the new mas-tekton container.

That role then, needs to donwload the pipelinerun from artifactory (that is why the artifactory key). Also, the task running inside that pipeline runs mas-tekton container, which lives in artifactory (that is why the registry). Considering we use the param list to automate also the different taskdefs params inside the upgrade-test pipeline and even the task's params and stepTemplate envs (and so pass that to run-pipeline) task, I thought would make sense to have those settings in all the pipelines (just to simplify automation, as these infos will not be used by the pipeline itself, but by the runpipeline task that will eventually run them).

What I could do here is to rename those attributes to remove all these `fvt_` prefixes for the items in common to avoid confusion (and add some comments about why they are there). What do you think?

Reply:
Having spent more time working on this PR today, I think the confusion comes from the naming of this ... it should really be another pair of `devops_xxx` variables, their purpose is all about defining the records in the database that are created/updated to record the results of the tekton actions.